topics/assembly/tutorials/ERGA-post-assembly-QC/tutorial.md

```diff
@@ -17,7 +17,8 @@ contributors:
 - GitFab93
 - gallardoalba
 - tbrown91
-
+tags:
```

Comment:
I don't know where this tag came from and why it is plants. Should I change it to something else?

Reply:
At the moment, this is what the user sees when they search for the tutorial:

<img width="1125" alt="image" src="https://github.com/user-attachments/assets/86a0cb35-61ec-49b4-b379-45ee99aae6f0">

Reply:
This tutorial can be used on plant genomes? Maybe add some more tags?